homeworks/hw_3.py

```diff
@@ -0,0 +1,79 @@
+"""
+Please wrap all logic in functions:
+1. Заменить в произвольной строке согласные буквы на гласные.
+"""
+import random
+consonants = ["b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "y", "z"]
+vowels = ["a", "e", "i", "o", "u", "y"]
```

Comment:
during the code, those variables will never be changed(remove letters or add letters to those lists), based on that we could use one of mutable data types(str() or tuple())
`vowels = ("a", "e", "i", "o", "u", "y"), consonants = ("b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "y", "z")`